### PR TITLE
upgrade edit form

### PR DESCRIPTION
**Pull request by NewIvanIvanov**

### CODE REVIEW

File: client/apps/city_issues/forms/forms.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    """Edit issue form."""
+    name = CharField(
+        min_length=5,
+        max_length=50,
+        widget=TextInput({'size': 50}),
+        required=True)
+
+    description = CharField(
+        min_length=5,
+        max_length=50,
```
Comment (KostSvitlana) [inline client/apps/city_issues/forms/forms.py]: I think `max_length=50` is not enough for description

File: client/apps/city_issues/views.py [hunk 2]
```diff
@@ -4,10 +4,12 @@
 # -*- coding: utf-8 -*-
 from django.core import serializers
 from django.http import JsonResponse
-from django.shortcuts import render
+from django.shortcuts import render, redirect
```
Comment (KostSvitlana) [inline client/apps/city_issues/views.py]: not by alphabet

File: client/basic/urls.py [hunk 3]
```diff
@@ -17,7 +17,8 @@
 
 from city_issues.views import (
     HomePageView, get_issue_data,
-    map_page_view, get_all_issues_data)
+    map_page_view, get_all_issues_data,
+    edit_issue_view)
```
Comment (KostSvitlana) [inline client/basic/urls.py]: not by alphabet

File: client/apps/city_issues/forms/forms.py (hunk 1 above)
Comment (alevitsky) [reply client/apps/city_issues/forms/forms.py]: for the test version is enough

**Review by KostSvitlana — APPROVED** [review]

**Review by alevitsky — APPROVED** [review]

**Pull request merged**